tests/_test_mscolab/test_files_api.py

```diff
@@ -172,8 +171,6 @@ def test_get_all_changes(self):
         with self.app.test_client():
             flight_path, operation = self._create_operation(flight_path="V11")
             assert self.fm.save_file(operation.id, "content1", self.user)
-            # we need to wait to get an updated created_at
-            time.sleep(1)
```

Comment:
_Technically_ it would still need to wait a microsecond. Practically, I don't think that will ever be an issue...

Reply:
testing with sqlite would need a millisecond.